src/client/fuse_ll.cc

```diff
@@ -238,6 +240,8 @@ static void fuse_ll_lookup(fuse_req_t req, fuse_ino_t parent, const char *name)
   if (r >= 0) {
     fe.ino = cfuse->make_fake_ino(fe.attr.st_ino, fe.attr.st_dev);
     fe.attr.st_rdev = new_encode_dev(fe.attr.st_rdev);
+    fe.entry_timeout = cfuse->entry_timeout;
+    fe.attr_timeout = cfuse->attr_timeout;
```

Comment:
I think you need to set the timeouts in other places where are setting the `fuse_entry_param` parameter.

Reply:
You didn't fix this yet with your new commit.

They are in somewhere like `fuse_ll_mknod()`, `fuse_ll_mkdir()`, etc.

Reply:
Or just add a new helper like:

```
static void fuse_entry_param_init(CephFuse::Handle *cfuse, struct fuse_entry_param* fe)
{
    fe.entry_timeout = cfuse->entry_timeout;
    fe.attr_timeout = cfuse->attr_timeout;
}
```

Call this in mknod/mkdir/lookup,etc.

Reply:
Thank you. I'll fix it soon.